dev_requirements

```diff
@@ -9,4 +9,5 @@ black
 wheel==0.30.0
 pre-commit
 pylint
-flake8
\ No newline at end of file
+flake8
+azure-storage-blob
```

Comment:
Where is the xdist plugin defined?

Reply:
Yeah x-dist should be added here, will make the update